build.gradle

```diff
@@ -152,11 +152,11 @@ configurations {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:4.2.0')
-    api 'com.exactpro.th2:grpc-common:4.2.0-dev'
+    api platform('com.exactpro.th2:bom:4.4.0-update-libs-5345334944-c71fdc9-SNAPSHOT')
```

Comment:
Will we update them to the releases before merging PR?

Reply:
Yes, we should